Dockerfile

```diff
@@ -86,6 +88,17 @@ RUN git clone https://github.com/codemasher/php-ext-xz.git --branch ${PHP_EXT_XZ
   && phpize \
   && ./configure \
   && make && make install
+  
+## Zopfli Extension
+FROM compile AS zopfli
+RUN git clone --recursive https://github.com/kjdev/php-ext-zopfli.git \
+  && cd php-ext-zopfli \
+  &&sudo apt-get update \
+  &&sudo apt-get install zlib1g-dev \
```

Comment:
1. This is not how packages are installed in alpine
2. Required packages should be installed in the `RUN apk add` section above

```suggestion
```